bindings/java/src/org/sleuthkit/datamodel/IngestStatusType.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.datamodel;
+
+enum IngestStatusType {
+
+	STARTED(0, "Started"),
+	CANCELLED(1, "Cancelled"),
+	COMPLETED(2, "Completed");
+	private int typeId;
+	private String typeName;
+
```

Comment:
Let's just use the enum value instead of a typeID field. We can also use name of the enum values for the typeName. There will be no need for getters for these. Let's have displayName field with a getter,  and the put the display name in the database.